ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -339,6 +339,9 @@ public struct Avatar: View, TokenizedControlView {
                                     .overlay(accessoryImage
                                         .interpolation(.high)
                                         .resizable()
+                                         // Rendering mode should be original for now to allow any image to show,
+                                         // but eventually should be changed to template when we add foreground tokens.
+                                        .renderingMode(shouldDisplayPresence ? .template : .original)
```

Comment:
i feel like we should respect accessoryImage.renderingMode? just because it is cutout doesn't mean it has to be original

Reply:
Agreed, and it was going to be a change in a follow-up PR for the following reasons:
- Presence should always be template if we want to stick to FluentUI design -- anything other than template will look like this: <img width="81" alt="image" src="https://user-images.githubusercontent.com/22566866/217145119-d50c210b-126e-46c9-98f9-2c5b0f5fe641.png">
- Following the accessoryImage.renderingMode(<insert rendering mode here>) just means I need to expose another API anyway, which is what we want to avoid in this change here. This was going to be introduced in the next change where I can expose a property that allows user to set which rendering mode they'd like to use

Reply:
what is the exposed api? isn't it just part of UIImage?

Reply:
Synced offline -- I was talking about the SwiftUI context, not the UIKit context. Setting the property on the UIKit side will reduce over-complicating logic in the component side